src/routes/volume/VolumeActions.js

```diff
@@ -24,21 +24,14 @@ function actions({ selected, engineImages, showAttachHost, detach, showEngineUpg
         })}</div>
       </div>)
     }
-    if (record.state === 'attached') {
```

Comment:
We should keep the case of the user being warned about the volume being attached. Otherwise, the user may be attaching and using the volume without Kubernetes but not get a warning since there are no workloads and encounter errors. The attached case and the workload case should be separate.